src/test/java/ezvcard/io/roundtrip/XCardRoundTripTest.java

```diff
@@ -0,0 +1,82 @@
+package ezvcard.io.roundtrip;
+
+import java.io.*;
+
+import org.junit.Test;
+
+import ezvcard.VCardVersion;
+import ezvcard.io.StreamReader;
+import ezvcard.io.StreamWriter;
+import ezvcard.io.xml.XCardReader;
+import ezvcard.io.xml.XCardWriter;
+
+public class XCardRoundTripTest extends RoundTripTestBase {
+
+	public XCardRoundTripTest() throws Exception {
+		updateSamples(VCardVersion.V4_0,
+				"outlook-2003" // &#12; in fburl is not valid xml
+		);
+		updateSamples(VCardVersion.V3_0);
+	}
+
+	@Test
+	public void equals_compare_vcard_4_to_xcard() throws Exception {
+		convertAllFromVCard(VCardVersion.V4_0, true, false,
+				"outlook", // newlines not preserved on linux
```

Comment:
Thank you for the clarification.  I see what you're saying.  Yes, the `=0D=0A` is being converted to `\n`, which represents the local system's newline character sequence.

While this is technically data loss, I think it's overkill to classify it as such.  The main reason `=0D=0A` is being used is because vCard version 2.1 does not support the `\n` escape sequence.  Therefore, if you want to encode newline characters in a property value using this vCard version, you must use quoted-printable encoding.

I think it's safe to say that `=0D=0A` can be interpreted generically as "newline sequence", so its conversion to `\n` isn't a problem.